packages/lintcstubs-arity/lintcstubs-arity.0.2.0/opam

```diff
@@ -0,0 +1,41 @@
+opam-version: "2.0"
+synopsis: "Generate headers for C bindings"
+description:
+  "Generates .h files from 'external' declarations in .ml or .cmt files. Can be used to spot mismatches in number of arguments between C primitive declared in OCaml and its implementation in the .c file."
+maintainer: ["Edwin Török <edwin.torok@cloud.com>"]
+authors: ["Edwin Török <edwin.torok@cloud.com>"]
+license: "LGPL-2.1-or-later"
+homepage: "https://github.com/edwintorok/lintcstubs-arity"
+bug-reports: "https://github.com/edwintorok/lintcstubs-arity/issues"
+depends: [
+  "dune" {>= "2.7"}
```

Comment:
This is probably a little low as it fails with 2.7.0 over an `(enabled_if )` construct

Reply:
Looks like this doesn't work on any 2.x, bumped it to 3.0 for now. I'll come back to this later to see whether I can make it work on 2.x (maybe by separating the package based on the OCaml version it supports).